### PR TITLE
Tests for Alerts.tsx

### PR DESCRIPTION
**Pull request by yassineejjoud**
Tests for CommonUI/src/Components/Alerts/Alert.tsx

image

### CODE REVIEW

File: CommonUI/src/Tests/Alerts.test.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                type={Alertoptions.type}
+                title={Alertoptions.title}
+                strongTitle={Alertoptions.strongTitle}
+            />
+        );
+        const testInstance: ReactTestInstance = testRenderer.root;
+
+        expect(
+            testInstance
+                .findAllByType('div')[2]
```
Comment (simlarsen) [inline CommonUI/src/Tests/Alerts.test.tsx:60]: You can id this div instead of array[2]

File: CommonUI/src/Tests/Alerts.test.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    };
+    const testRenderer: ReactTestRenderer = renderer.create(
+        <Alert
+            type={Alertoptions.type}
+            title={Alertoptions.title}
+            strongTitle={Alertoptions.strongTitle}
+        />
+    );
+    const testInstance: ReactTestInstance = testRenderer.root;
+    const allElements = testInstance
```
Comment (simlarsen) [inline CommonUI/src/Tests/Alerts.test.tsx:98]: Please do not use array[indexes], you can id the div.

**Review by simlarsen — CHANGES_REQUESTED** [review]

File: CommonUI/src/Tests/Alerts.test.tsx (hunk 1 above)
Comment (yassineejjoud) [reply CommonUI/src/Tests/Alerts.test.tsx:60]: Okay I will take care of that.

File: CommonUI/src/Tests/Alerts.test.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    const testRenderer: ReactTestRenderer = renderer.create(
+        <Alert
+            type={Alertoptions.type}
+            title={Alertoptions.title}
+            strongTitle={Alertoptions.strongTitle}
+        />
+    );
+    const testInstance: ReactTestInstance = testRenderer.root;
+
+    expect(testInstance.findAllByType('strong')[0]?.props['children']).toEqual(
```
[the remark this answers is not in the document]
Comment (yassineejjoud) [reply CommonUI/src/Tests/Alerts.test.tsx]: done

File: CommonUI/src/Tests/Alerts.test.tsx (hunk 2 above)
Comment (yassineejjoud) [reply CommonUI/src/Tests/Alerts.test.tsx]: done

**Comment by simlarsen** [comment]
This PR is abandoned.

**Pull request closed without merging**